xdp-filter/tests/test-xdp-filter.sh

```diff
@@ -261,6 +261,64 @@ test_print()
     check_run $XDP_FILTER unload $NS -v
 }
 
+
+get_python()
+{
+    if [[ -z "${PYTHON:-}" ]]; then
+        local -a possible=(python3 python)
+        local -a available
+
+        available=($(which --skip-alias --skip-functions "${possible[@]}" 2> /dev/null))
+        if [[ $? == "${#possible[@]}" ]]; then
```

Comment:
Here you're comparing the return code to the number of elements in the array? With a string comparison? Shouldn't it just be `$? -ne 0` like below?

Reply:
The exit code is the number of failed arguments, so I am checking whether it failed to find them all.

Reply:
Ah, I see. You're really bringing out all the esoteric shell features here, huh? ;)

Please add a comment explaining this; and it should be `-eq` instead of `==` for numeric comparison :)